vendor/modules.txt

```diff
@@ -144,6 +144,9 @@ github.com/containers/image/v5/signature
 github.com/containers/image/v5/transports
 github.com/containers/image/v5/types
 github.com/containers/image/v5/version
+# github.com/containers/kubensmnt v1.1.2
+## explicit; go 1.18
+github.com/containers/kubensmnt/utils/systemd
```

Comment:
it's not clear to me, why do we need this vendored here?

Reply:
I'm using my go-embed-vendor trick to get the systemd service (kubens.service) and drop-ins from github.com/containers/kubensmnt to copy in to the templates directory (the copy is done by the `hack/update-templates.sh` script).

Reply:
ah right thanks!